raiden/ui/cli.py

```diff
@@ -308,9 +311,31 @@ def app(address,
         print(e.message)
         sys.exit(1)
 
+    discovery_gasprice = GAS_PRICE
+    discovery_startgas = GAS_LIMIT
+    discovery_tx_cost = discovery_gasprice * discovery_startgas
+
+    while True:
```

Comment:
The `app` function is getting too large and confusing, please, make a new function to unlock the account, it should:
- Ask the user which account to use
- Do the required check for that account (available funds)
- Return the account data